### PR TITLE
Feature es6

### PR DESCRIPTION
**Pull request by vikipretium**
> ## Description
> Hi! wave In today's PR I recreated the **Awesom Books** project using `JavaScript ES6`.
> 
> ### General requirements
> * [x]  There are no linter errors.
> * [x]  Followed the **Github flow**.
> * [x]  The documentation is professional.
> 
> ### HTML/CSS and JavaScript requirements
> * [x]  Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
> * [x]  Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
> 
> ### Project Requirements
> * [x]  Setup a new project using the provided format.
> * [x]  Re-created the _Awesome books_ app following the rules:
>   
>   * Use ES6 syntax.
>   * The code is divided into multiple modules.
>   * All the functionalities of the project work correctly.
> * [x]  Displayed the current date below the navigation bar using **luxon**.
> 
> Thanks for taking the time to review my project.

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  navlist, navadd, navcon, listfun, addfun, confun,
+} from '../modules/nav.js';
+
+// eslint-disable-next-line no-unused-vars
+// const displayTime = () => {
+//   document.getElementById('current-date').innerText = new Date().toLocaleString();
+//   setTimeout(displayTime, 1000);
+// };
+// displayTime();
+
```
Comment (elisha2kyakpo1) [inline assets/js/app.js]: - [ ] Let's remove the commented code, it's part of making our codebase look clean best practice.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @vikipretium,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
- The project PR looks professional(title and PR description) ✔️
- The readme file is well written 👍
- All action checks aka linters are passing 👍
- The project works as expected ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/app.js (hunk 1 above)
Comment (vikipretium) [reply assets/js/app.js]: Thanks for reviewing my project @elisha2kyakpo1 :pray: 
I'll remove these comments.

**Review by Lameck1 — APPROVED** [review]
Hi, @vikipretium,

The tasks for this milestone have been successfully implemented 💯.
All the best with the next milestone.

You are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**